docs/bootstrap.md

```diff
@@ -46,61 +46,62 @@ import { resolvers } from "./resolvers";
 const schema = await buildSchema({ resolvers });
 ```
 
-There are also other options related to advanced features like [authorization](authorization.md) or [validation](validation.md) - you can read about them in docs.
+There are also other options related to advanced features like [authorization](./authorization.md) or [validation](./validation.md) - you can read about them in docs.
```

Comment:
Does the linking still works in docusaurus? I remember some old version I was using that time required having direct file names instead of relative paths.